user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -326,6 +326,9 @@ From DataMiner 10.2.0/10.1.12 onwards, users authenticated by Azure AD using SAM
 
 1. Restart the DataMiner Agent.
 
+> [!NOTE]
+> If your default username is not in email format or if DataMiner is unable to locate it, please configure the *\<PreferredEmailClaim>* tag to ensure it points to the correct email address.
+
```

Comment:
Per your request, I have added this as a note to the general section on automated user creation, @MichielMN.